src/website/assets/scripts/docs-expansion-panel.js

```diff
@@ -0,0 +1,42 @@
+
+document.addEventListener('DOMContentLoaded',
+  function() {
+
+    const ep = chi.expansionPanel(document.getElementById('example1'));
+
+    Array.prototype.forEach.call(
+      document.querySelectorAll('input[name="example1"]'),
+      function(input) {
+        input.addEventListener('change', function() {
+          if (this.checked) {
+            ep.setState(this.value);
+          }
+        }, false);
+      }
+    );
+
+    chi.expansionPanel(document.getElementById('example2'));
+    chi.expansionPanel(document.querySelectorAll('[data-chi-ep-group="example3"]'));
+    chi.expansionPanel(document.querySelectorAll('[data-chi-ep-group="example4"]'), {mode: 'accordion'});
+    chi.expansionPanel(document.querySelectorAll('[data-chi-ep-group="example5"]'), {mode: 'free'});
+    chi.expansionPanel(
+      document.querySelectorAll('[data-chi-ep-group="example6"]'),
+      {
+        mode: 'custom',
+        changeHandler: function (newState, oldState, expansionPanel, panelGroup) {
+          if (newState === chi.EXPANSION_PANEL_STATES.ACTIVE) {
+            panelGroup.expansion_panels.forEach(function (exPa) {
+              if (exPa !== expansionPanel) {
+                exPa.setState(chi.EXPANSION_PANEL_STATES.DONE.NAME);
+              }
+            });
+          }
+        }
+      }
+    );
+
+    chi.tab(document.querySelectorAll('.a-tabs'));
```

Comment:
@dani-cl-madrid documentation sidenav tabs are no longer functioning

Reply:
@mattnickles My mistake. You catched it! This has given me an idea for moving the sliding border when a user moves to another section. Maybe you remember I told you I couldn't do it. It is quite easy indeed. Just assign me a task if you want me to do it. If this happens, I will have a question: should I add support for sliding border for tabs in the second level? Thank you!

Reply:
Nice! I'll open a ticket for it.